custom_checks/custom_check_1.py

```diff
@@ -0,0 +1,18 @@
+import logging
```

Comment:
Maybe rename the file to custom_check_sample.py to make it clear?

Reply:
Renamed it.